bounties/npm/fsa/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-09-02",
+	"AffectedVersionRange": "*",
+	"Summary": "Command Injection",
+	"Contributor": {
+		"Discloser": "",
+		"Fixer": ""
+	},
+	"Package": {
+		"Registry": "npm",
+		"Name": "fsa",
+		"URL": "https://www.npmjs.com/package/fsa",
+		"Downloads": "48454"
+	},
+	"CWEs": [
+		{
+			"ID": "CWE-79",
+			"Description": ""
+		}
+	],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "L",
+		"AC": "H",
+		"PR": "L",
+		"UI": "N",
+		"S": "U",
+		"C": "L",
+		"I": "L",
+		"A": "N",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "3.6"
```

Comment:
I would higher the severity since it's an `arbitrary command injection`

Reply:
I've had a look and at the moment agree mostly with that CVSS - I'd bump it to 3.9 tops once it's weighted but think it should be OK as 3.6